rules/macos/privilege_escalation_explicit_creds_via_apple_scripting.toml

```diff
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2020/12/07"
+maturity = "production"
+updated_date = "2020/12/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies execution of the security_authtrampoline process via the Apple script interpreter (osascript). This occurs
+when programs use AuthorizationExecute-WithPrivileges from the Security.framework to run another program with root
+privileges. It should not be run by itself, as this is a sign of execution with explicit logon credentials.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Execution with Explicit Credentials via Apple Scripting"
+references = [
+    "https://objectivebythesea.com/v2/talks/OBTS_v2_Thomas.pdf",
+    "https://www.manpagez.com/man/8/security_authtrampoline/",
+]
+risk_score = 47
+rule_id = "f0eb70e9-71e9-40cd-813f-bf8e8c812cb1"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Execution", "Privilege Escalation"]
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan = 5s
+ [process where event.type in ("start", "process_started", "info") and process.name : "osascript"] by process.pid
+ [process where event.type in ("start", "process_started") and process.name : "security_authtrampoline"] by process.ppid
```

Comment:
```suggestion
sequence by host.id with maxspan=5s
 [process where event.type in ("start", "process_started", "info") and process.name == "osascript"] by process.entity_id
 [process where event.type in ("start", "process_started") and process.name == "security_authtrampoline"] by process.parent.entity_id
```

Reply:
same as others (replaced entity_id with pid or ppid, not yet in auditbeat)